src/features/layout/NavBar.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import PublicLinks from '../../constants/Links';
+
+function Navbar() {
```

Comment:
should be under layouts, not under features layouts is another folder in src

Reply:
Okay